tests/Extension/SymfonyTest.php

```diff
@@ -39,4 +39,11 @@ public function testAssertJsonValueEquals()
 
         Symfony::assertJsonValueEquals(123, 'foo', $response);
     }
+
+    public function testAssertJsonResponse()
+    {
+        $response = new Response('{}', 200, ['Content-Type' => 'application/json']);
+
+        Symfony::assertJsonResponse($response);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\EnricoStahn\JsonAssert\Extension\Symfony' in method 'testAssertJsonResponse'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2281671867/issues/source?bid=3181605&fileBranchId=3188493#l47)